tools/doms-data-tools/update_doms_data_pk.py

```diff
@@ -0,0 +1,299 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Can you add a docstring explaining at a high level what this should be used for? we can also mention it in the release notes.

Reply:
@RKuttruff Where did you resolve this comment? Otherwise it looks good to go and merge!

Reply:
@skorper 
I put it below the ASF header. I think the ASF header is required to be at the top of the file.

Reply:
Thanks! I missed that somehow